src/App.js

```diff
@@ -3,14 +3,28 @@ import Homepage from "./components/Homepage"
 import Header from "./components/Header"
 import Jumbotron from './components/Jumbotron'
 import Footer from "./components/Footer"
+import Order_page from "./components/Order_page";
```

Comment:
Please rename this component to be PascalCase instead - it's not a big deal, but the convention in react is to use PascalCase instead (OrderPage)

You should rename the component itself, the folder it's in and this import